oeps/oep-2.rst

```diff
@@ -0,0 +1,100 @@
+==========================
+OEP-2: Repository Metadata
+==========================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`oep-2`                              |
++---------------+-------------------------------------------+
+| Title         | Repository Metadata                       |
++---------------+-------------------------------------------+
+| Last-Modified | 2016-04-21                                |
++---------------+-------------------------------------------+
+| Author        | Calen Pennington <cale@edx.org>           |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Best Practice                             |
++---------------+-------------------------------------------+
+| Created       | 2016-04-21                                |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+Proposes a standard format for repository metadata for Open edX projects.
+
+Specification
+=============
+
+Each repo in the Open edX system will include a file ``openedx.yml``, with the
+following fields:
+
+``owner``: string
+    The github user/team responsible for the repository.
+
+``nick``: string
+    A short-name for the repository used by reporting tools.
+
+``area``: string
+    Used to group repositories by in reporting tools. Sample areas: core,
+    mobile analytics.
+
+``oeps``: dictionary
+    A list of keys corresponding to Best Practice OEPs (in the format
+    ``oep-n``). Each value would be one of ``True``, ``False``, ``auto`` or a
+    reason dictionary. ``True`` and ``False`` would indicate accordance (or
+    lack thereof) with the specified OEP, and ``auto`` would specify that the
+    tooling should auto-detect accordance with the OEP. A reason dictionary
```

Comment:
"auto" confuses me.  What kind of tooling are we envisioning? If it could determine it automatically, why does it need to be told to do that?